db/migrate/20170711193304_remove_accepted_from_users.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveAcceptedFromUsers < ActiveRecord::Migration
+  def change
+    remove_column :users, :accepted, :boolean
```

Comment:
I tried that and got an error. Should I delete these migrations that we don't need and then recommit? Will that destroy the world?